testing/scenario_app/ios/Scenarios/Scenarios/AppDelegate.m

```diff
@@ -9,6 +9,18 @@
 #import "ScreenBeforeFlutter.h"
 #import "TextPlatformView.h"
 
+// A UIViewController that sets YES for its preferedStatusBarHidden property.
```

Comment:
can you add a comment why we need this workaround.

Reply:
Done